XenModel/Messages.resx

```diff
@@ -7616,6 +7616,24 @@ SR UUID: {1}</value>
   <data name="LAUNCH_LICENSE_MANAGER" xml:space="preserve">
     <value>License Manager</value>
   </data>
+  <data name="LEAF_COALESCE_COMPLETED_DESCRIPTION" xml:space="preserve">
+    <value>Data merge for VM {0} completed</value>
+  </data>
+  <data name="LEAF_COALESCE_COMPLETED_TITLE" xml:space="preserve">

```

Comment:
From what I understood from the task's description, we don't want to show the leaf coalescing text as is, which means we either need to make the alert's Title return the same as the Description, or ask @MarkSymsCtx whether it night be useful for the user to also see the VDI's name in the detailed description (e.g. something like "Data is being merged for VM x on disk y"?).